lang/Omega/Portfile

```diff
@@ -11,22 +11,18 @@ long_description \
     providing type-level computations and \
     by this virtue integrates a theorem prover.
 
-homepage                http://web.cecs.pdx.edu/~sheard/Omega/
-master_sites            ${homepage}
-distname                Omega${version}b
-checksums               md5 b5e5505626a1d5bdc8c539ce54112946 \
-                        sha1 93752d7a24a1c6f9f1b21268c51726fd55e0f260 \
-                        rmd160 611b6c4870e647634488c96d9eae27bf22846ebc
+homepage                https://web.cecs.pdx.edu/~sheard/Omega/
+master_sites            googlecode:omega
```

Comment:
See https://trac.macports.org/ticket/53467

Reply:
I know that googlecode is no more, but couldn't find a better hosting spot for this. Like I said above: Its unclear if its worth keeping this port around.
That said, since it failed I'm debating just deleting the port.